src/ngMessages/messages.js

```diff
@@ -496,31 +496,16 @@ angular.module('ngMessages', [])
     * @param {string} ngMessagesInclude|src a string value corresponding to the remote template.
     */
    .directive('ngMessagesInclude',
-     ['$templateRequest', '$document', '$compile', function($templateRequest, $document, $compile) {
+     ['$templateRequest', '$compile', function($templateRequest, $compile) {
 
      return {
        restrict: 'AE',
-       require: '^^ngMessages',
-       compile: function(element, attrs) {
-         var comment = jqLite($document[0].createComment(' ngMessagesInclude: '));
-         element.after(comment);
```

Comment:
If we set terminal to true then will this run last if the priority is left unset?